appserver/web/jsf-connector/src/main/java/org/glassfish/faces/integration/GlassFishInjectionProvider.java

```diff
@@ -473,8 +474,8 @@ public void enableHighAvailability(ServletContext ctx) {
                 }
                 
                 if (isDistributable && enableHA) {
-                    LOGGER.fine("setting the EnableAgressiveSessionDirtying to true");
-                    config.overrideContextInitParameter(EnableAgressiveSessionDirtying, TRUE);
+                    LOGGER.fine("setting the EnableDistributable to true");
```

Comment:
Might as well fix the grammar
```suggestion
                    LOGGER.fine("setting EnableDistributable to true");
```